omnibus/config/software/chef-dk.rb

```diff
@@ -75,7 +75,7 @@
   appbundle "chef", lockdir: project_dir, gem: "chef", without: %w{integration docgen maintenance ci travis}, env: env
   appbundle "foodcritic", lockdir: project_dir, gem: "foodcritic", without: %w{development}, env: env
   appbundle "test-kitchen", lockdir: project_dir, gem: "test-kitchen", without: %w{changelog debug docs}, env: env
-  appbundle "inspec", lockdir: project_dir, gem: "inspec", without: %w{deploy tools maintenance integration}, env: env
+  appbundle "inspec-bin", lockdir: project_dir, gem: "inspec", without: %w{deploy tools maintenance integration}, env: env
```

Comment:
this needs to be `gem: "inspec-bin"`, i am not clear any more on what the first arg even does here....